bin/cron/with_pg/dms_tasks

```diff
@@ -27,6 +27,17 @@ SCHEMA_PERMISSIONS = {
   pegasus_pii: ['admin', 'reader_pii']
 }.freeze
 
+# @param task_name [String] The name of the task.
+# @param task_execution_time_sec [Integer] The most recent execution time (in seconds) for the task.
+# @return [Boolean] Whether the task should be run.
+def run_task?(task_name, task_execution_time_sec)
+  return false unless task_name.start_with? 'cron'
+  if task_execution_time_sec > 4500
+    return Time.now.saturday?
+  end
+  return true
+end
```

Comment:
Going to document here, adding a comment when the dust settles. Basically, we want to maximize data freshness while minimizing downtime and data transfer cost. This seems like a good tradeoff, though the approach and line (currently 4500 seconds) are both likely to change.